README.md

```diff
@@ -417,6 +422,18 @@ Stoplight.configure do |config|
 end
 ```
 
+#### DragonflyDB Support
+
+Although Stoplight does not officially support [DragonflyDB], it can be used with it. To achieve behavior similar to Redis, start DragonflyDB with the following flags:
+
+```sh
+./dragonfly --default_lua_flags=allow-undeclared-keys,disable-atomicity
```

Comment:
Doesn't it enforces a behavior the opposite to the Redis' default behavior?

Not sure about the behavior when the undeclared keys are allowed, but disabling atomicity makes Dragonfly's lua scripts behave more like Redis' pipelines - instructions from other clients can be executed between LUA Script's instructions making the begaviour completely unpredictable.

I suppose we should advise for bot using these flags, right?